Mobile Buy SDK/Mobile Buy SDK Tests/BUYClient+CustomerTests.m

```diff
@@ -197,29 +197,6 @@ - (void)testCustomerOrderWithID
 	[self waitForExpectationsWithTimeout:10.0 handler:^(NSError *error) {}];
 }
 
-#pragma mark - Update -
-
-- (void)testCustomerUpdate
```

Comment:
When running this against Shopify, shouldn't we check, for example, that changing the default address does the right thing? If we had two, we could alternate, although then the test wouldn't be stateless (unless we changed it and then changed it back).